### PR TITLE
WIP - Lintingding

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
Corrects all linting failures across our brands
EXCEPT for those failures which require more though and understanding of the project, for example css on ID's

WIP - May need version bumps.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
None

#### Screenshots / Screencast
None

#### What gif best describes how you feel about this work?
giphy-4

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/alton-towers/src/less/navbar.less [hunk 1]
```diff
@@ -107,14 +105,13 @@
     // Justified links in double navbar > primary navbar
     .navbar-primary {
       border-bottom-color: @hr-border; // Double nav separator
-      font-size: 0;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/navbar.less]: we should use this in our work. we have a class `sr-only` that does this.

File: brands/bcp/src/less/colours.less [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @color-blue-darker:  #0f2350;
 @color-blue-dark:    #193a83;
 @color-blue:         #70c5fe;
-@color-blue-light:   #1CA4FF;
+@color-blue-light:   #1ca4ff;
```
Comment (nickine) [inline brands/bcp/src/less/colours.less]: SO controversial - I must admit, I'm too used to seeing HEX uppercase but I'm really not bother either way

File: brands/budget-family-breaks/src/less/navbar.less [hunk 3]
```diff
@@ -87,10 +87,10 @@
...
   .navbar-container .navbar-primary,
   .navbar-container.navbar-primary .navbar-inverse:first-child {
-    padding-top: ceil(@navbar-height-sm - @navbar-height)/2;
-    padding-bottom: ceil(@navbar-height-sm - @navbar-height)/2;
+    padding-top: ceil(@navbar-height-sm - @navbar-height) / 2;
+    padding-bottom: ceil(@navbar-height-sm - @navbar-height) / 2;
     min-height: @navbar-height-sm;
-    box-shadow: 0 0 12px rgba(0, 0, 0, 0.175);
+    box-shadow: 0 0 12px rgba(0, 0, 0, .175);
     // margin-bottom: ceil(@navbar-height * 2);
```
Comment (nickine) [inline brands/budget-family-breaks/src/less/navbar.less]: @jordanhillier90 confirms that this commented out line can be deleted

File: brands/essentialtravel/src/less/buttons.less [hunk 4]
```diff
@@ -77,10 +77,10 @@
...
-  background: linear-gradient(to bottom, @topColor 0%,@topColor 50%,@bottomColor 51%,@bottomColor 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='@{topColor}', endColorstr='@{bottomColor}',GradientType=0 );
-}
\ No newline at end of file
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, @topColor), color-stop(50%, @topColor), color-stop(51%, @bottomColor), color-stop(100%, @bottomColor));
+  background: -webkit-linear-gradient(top, @topColor 0%, @topColor 50%, @bottomColor 51%, @bottomColor 100%);
+  background: -o-linear-gradient(top, @topColor 0%, @topColor 50%, @bottomColor 51%, @bottomColor 100%);
+  background: -ms-linear-gradient(top, @topColor 0%, @topColor 50%, @bottomColor 51%, @bottomColor 100%);
+  background: linear-gradient(to bottom, @topColor 0%, @topColor 50%, @bottomColor 51%, @bottomColor 100%);
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='@{topColor}', endColorstr='@{bottomColor}', GradientType=0);
```
Comment (nickine) [inline brands/essentialtravel/src/less/buttons.less]: kill it

File: gulpfile.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ gulp.task('less', function() {
 });
 
 gulp.task('lint', () => {
-  return gulp.src('./brands/*/src/less/*.less')
+  return gulp.src('./brands/warwick/src/less/*.less')
```
Comment (nickine) [inline gulpfile.js]: nononononono
nonono_cat

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/essentialtravel/src/less/variables.less [hunk 6]
```diff
@@ -218,7 +218,7 @@
 //
 // Carousel
 // -------------------------
-@carousel-text-shadow:                        0 0 0 rgba(0,0,0,0);
+@carousel-text-shadow:                        0 0 0 rgba(0, 0, 0, 0);
```
Comment (jodiedoubleday) [inline brands/essentialtravel/src/less/variables.less]: yes!

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/airparks/src/less/lists.less [hunk 7]
```diff
@@ -15,7 +15,7 @@
   li {
     display: table-cell;
     vertical-align: middle;
-    flex:1;
+    flex: 1;
```
Comment (nickine) [inline brands/airparks/src/less/lists.less]: Yay, Space

File: brands/thorpe-park/src/less/navbar.less [hunk 8]
```diff
@@ -52,18 +52,20 @@
...
       > a {
-        color:@navbar-default-link-color;
-        &:hover, &:active, &:focus {
-          color:@navbar-default-link-hover-color;
-          background:@navbar-default-link-hover-bg;
+        color: @navbar-default-link-color;
+        &:hover,
+        &:active,
+        &:focus {
+          color: @navbar-default-link-hover-color;
```
Comment (nickine) [inline brands/thorpe-park/src/less/navbar.less]: <3 readability

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request closed without merging**